akka-docs-dev/rst/scala/code/docs/http/scaladsl/server/ExceptionHandlerExamplesSpec.scala

```diff
@@ -2,45 +2,48 @@
  * Copyright (C) 2009-2014 Typesafe Inc. <http://www.typesafe.com>
  */
 
-package docs.http.server
+package docs.http.scaladsl.server
 
-import akka.actor.ActorSystem
-import akka.http.scaladsl.server.Route
-import akka.stream.ActorFlowMaterializer
+// format: OFF
 
-object MyHandler {
-  //# example-1
-  import akka.http.scaladsl.model.HttpResponse
-  import akka.http.scaladsl.model.StatusCodes._
+object MyExceptionHandler {
+
+  //#custom-handler-example
+  import akka.actor.ActorSystem
+  import akka.stream.ActorFlowMaterializer
+  import akka.http.scaladsl.Http
+  import akka.http.scaladsl.model._
   import akka.http.scaladsl.server._
+  import StatusCodes._
   import Directives._
 
   implicit def myExceptionHandler =
```

Comment:
show the non-implicit version first, then you can show the simplifyed version as an alternative. It is easy to get confused when learning a new library with implicits involved.